src/components/PackageCard/Highlight.tsx

```diff
@@ -0,0 +1,40 @@
+import { Stack, Image, Text } from "@chakra-ui/react";
+import type { FunctionComponent } from "react";
+import { OFFICIAL_SCOPES } from "../../constants/constructs";
+import { usePackageCard } from "./PackageCard";
+
+const HIGHLIGHT_RENDER_MAP = {
+  official: {
+    imgsrc: "/assets/construct.png",
+    label: "Official",
+    color: "#ED3B00",
+  },
+  community: {
+    imgsrc: "/assets/community.png",
+    label: "Community",
+    color: "#2F50FE",
+  },
+  highQuality: {
+    imgsrc: "/assets/medal.png",
+    label: "High Quality",
+    color: "#1F02D4",
+  },
+} as const;
+
+// TODO: We will need a similar solution to tags for determining High Quality, and likely Official
+export const Highlight: FunctionComponent = () => {
+  const { name } = usePackageCard();
+
+  const isOfficial = OFFICIAL_SCOPES.some((scope) => name.includes(scope));
```

Comment:
We can probably refactor this into a function in some shared function or hook in a later PR